src/libraries/Microsoft.Extensions.Logging.Abstractions/gen/LoggerMessageGenerator.Parser.cs

```diff
@@ -437,12 +437,20 @@ public IReadOnlyList<LoggerClass> GetLogClasses(IEnumerable<ClassDeclarationSynt
                                             // determine the namespace the class is declared in, if any
                                             SyntaxNode? potentialNamespaceParent = classDec.Parent;
                                             while (potentialNamespaceParent != null &&
-                                                   potentialNamespaceParent is not NamespaceDeclarationSyntax &&
-                                                   potentialNamespaceParent is not FileScopedNamespaceDeclarationSyntax)
+                                                   potentialNamespaceParent is not NamespaceDeclarationSyntax
```

Comment:
I think the latest version is using `BaseNamespaceDeclarationSyntax`

see https://github.com/dotnet/runtime/pull/57894/files

Reply:
These changes are sync'd to that change. Note that in the #57894 change, line 447 uses `BaseNamespaceDeclarationSyntax`, but lines 442-443 use `NamespaceDeclarationSyntax` and `FileScopedNamespaceDeclarationSyntax`, as this change does.